system/Router/RouteCollection.php

```diff
@@ -233,7 +234,7 @@ class RouteCollection implements RouteCollectionInterface
 	 * @param \CodeIgniter\Autoloader\FileLocator $locator
 	 * @param \Config\Modules                     $moduleConfig
 	 */
-	public function __construct(FileLocator $locator, $moduleConfig)
+	public function __construct(FileLocator $locator,Modules $moduleConfig)
```

Comment:
For coding standard, it need space between `,` and `Modules`, you can run `composer update` locally, so the `bash admin/setup.sh` will be executed so next change will validate coding standard first before commit.